ouroboros-consensus-cardano/changelog.d/20231002_154149_armandoifsantos_fix_peerSharing_handshake.md

```diff
@@ -0,0 +1,20 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+### Non-Breaking
+
+- Changes `NoPeerSharing` to `PeerSharingDisabled` ([#4644](https://github.com/input-output-hk/ouroboros-network/pull/4644))
```

Comment:
- `PeerSharingDisabled` was renamed to `NoPeerSharing` upstream. I don't think that needs to be mentioned in the Consensus change log.
- On the other hand, I would imagine it does mean that the `build-depends` field should be updated with an greater lower bound?

Reply:
Yes, but should I do that before network patches get a release?

Reply:
I think so, but I don't know if any such policy is technically established.

A related thought: the PR cannot merge until CI passes, and CI won't pass until such a network release is on CHaP.

Reply:
I think that is alright, the PR is small and can wait to be merged until everything is in place

Reply:
Ouroboros-network is released, and I modified the changelog fragments. I will dismiss this request for changes.